web/client/components/misc/quillmodules/ResizeModule.js

```diff
@@ -723,6 +723,7 @@ module.exports = Quill => {
             iframe.setAttribute('height', height);
             let overlay = document.createElement('DIV');
             overlay.setAttribute('class', 'ms-quill-iframe');
+            overlay.style.display = 'none';
```

Comment:
Where is the overlay displayed again?

Reply:
The overly (div) is rendered on top of the video, so the user cannot click to play it. It is a different behavior than what @offtherailz have described, but could be related

Reply:
... with this change is created not visible. Is there any piece of code that makes this div visible? I cannot find it

Reply:
Setting display to hidden makes the div non-interactive, so the user cannot click on it. but it can be selected by its ID or class. As I understand, that is the purpose of the div( resize the div and hence the iframe)